README.md

```diff
@@ -1,79 +1,97 @@
-also supports event-driven communication with sc-memory and sc-agents, located in it.
-
-Start use [OSTIS-web-platform](https://github.com/ostis-ai/ostis-web-platform) to integrate all the world solutions ***by 
-the way of the same technology***, ***the same data representation and programming language*** and ***the same program 
-software***!
-
-# Documentation: 
-
-- A part of sc-machine documentation, represented in md format: [GitHub Pages](https://ostis-ai.github.io/sc-machine).
-  It serves as a guide for installation, development and a brief description of the project modules.
-- SCn-code represented sc-machine documentation: [LaTeX documentation](https://github.com/ostis-ai/ostis-web-platform/blob/develop/docs/main.pdf).
-  This documentation fully describes the sc-machine: its components, software interfaces for interacting with sc-memory, 
-  the characteristics and design of sc-memory, as well as the rationale for the sc-machine tools and solutions used.
-
-# Quick start
-Sc-machine is a core of any ostis-system and a part of on of them, named **OSTIS Web platform**.
-You can use our [OSTIS Web platform](https://github.com/ostis-ai/ostis-web-platform) project to get up and running quickly.
-# Installation
-## Docker
-We're providing a Docker image for this project. Head to [Installing with Docker](https://ostis-ai.github.io/sc-machine/docker) 
-section of our docs to try it out! We recommend to use this way for sc-machine deployment.
-## Native
-If you do not have the opportunity to deploy the system using docker, please, refer to the docs for your OS: 
-[Build on Linux](https://ostis-ai.github.io/sc-machine/build/linux-build/) or [Build on macOS](https://ostis-ai.github.io/sc-machine/build/osx-build/).
-Nowadays sc-machine isn't *natively* support on Windows.
-
-# Usage
-Typical usage case is to install platform using Docker or by building from sources, build knowledge base and start 
-**sc-server** for other apps. By default, sc-server is runnable on host 127.0.0.1 and port 8090. Before sc-server 
-running, sure, that all needed SCs and SCg sources is located in kb folder.
-
-To run sc-server, you can do the following:
-```sh
-#build kb
-python3 scripts/build_kb.py -c config/sc-machine.ini <path to kb folder with SCs and SCg sources or repo.path file>
-#launch server
-./bin/sc-server -c config/sc-machine.ini
-``` 
-
-You can use `healthcheck.py` script to check memory state in sc-server.
-```sh
-python3 scripts/healthcheck.py --host <sc-server host> --port <sc-server port>
-``` 
-
-To get more information about these scripts use `-h` flags upon their launch.
+Short version: **sc-machine** is an agent-based knowledge graph processing machine.
+
+The knowledge graphs are represented in SC-code (language of universal knowledge representation) provided by the [**OSTIS Technology**](https://github.com/ostis-ai/ostis-project).
```

Comment:
```suggestion
Sc-machine stores and process the knowledge graphs represented in SC-code (language of universal knowledge representation) and is used to develop, design and exercise new generation computer systems. All ontology-based design of sc-machine is provided by the [**OSTIS Technology**](https://github.com/ostis-ai/ostis-project).
```

Reply:
`ontology-based design of sc-machine` - what do you mean by that?
`exercise new generation computer systems` - ...and that.
I agree that a bit more info should be added, but I am not sure what did you have in mind with these two. My current version would be something along those lines:
```suggestion
sc-machine stores and processes knowledge graphs represented in SC-code (language of universal knowledge representation) provided by the [**OSTIS Technology**](https://github.com/ostis-ai/ostis-project).
```

Reply:
We can discuss this and I told you about main ideas of sc-machine in key in which linked concepts are formalized in the OSTIS Standard

Reply:
After discuss
```suggestion
SC-machine stores and processes knowledge graphs represented in SC-code (language of universal knowledge representation). All ontology-based design of sc-machine is provided by the [**OSTIS Technology**](https://github.com/ostis-ai/ostis-project).
```

Reply:
After discuss: apply Pavel's variant from this conversation